tools/nogo.js

```diff
@@ -1,8 +1,307 @@
 {
   "checkunsafe": {
     "exclude_files": {
-      "/external/": "not subject to constraint"
+      "/external/": "allowed: not subject to unsafe naming rules"
     }
   },
-  "noescape": {}
+  "composites": {
+    "exclude_files": {
+      ".*_test.go": "allowed: allow for convenience in tests",
+      "/external/": "allowed: not subject to style constraint",
+      "/pkg/metric/metric.go": "allowed: verbose protos",
+      "/pkg/sentry/socket/rpcinet/.*": "allowed: verbose protos",
+      "/pkg/state/(encode|decode).go": "allowed: verbose protos",
+      "/pkg/sentry/devices/memdev/random.go": "fix: gvisor.dev/gvisor/pkg/safemem.FromIOReader composite literal uses unkeyed fields",
```

Comment:
Make sure we file a bug(s) to get these fixed.

Reply:
Sure, but I'd like to do that after they are in place, since so many can be fixed in a trivial sweep, we may just want bugs for those that will be around for a while.